Doc/library/curses.rst

```diff
@@ -988,7 +988,10 @@ the following methods and attributes:
             window.getstr(y, x, n)
 
    Read a bytes object from the user, with primitive line editing capacity.
+   The maximum value for *n* is 2047.
 
+   .. versionchanged:: next
+      The maximum value for *n* is set to 2047.
```

Comment:
```suggestion
   .. versionchanged:: next
      The maximum value for *n* is set to 2047.

```

Sorry, we actually separate functions by two blank lines... Thanks for your patience by the way.